src/gameObjects/map_start/index.js

```diff
@@ -3,7 +3,13 @@ import { cake } from './cake.gameObject';
 import { npcsOnChairs } from './npcsOnChairs.gameObject';
 import { ztmTrailer } from './ztmTrailer.gameObject';
 import { computer } from './computer.gameObject';
+<<<<<<< HEAD
```

Comment:
Please remove the merge conflict comments and double check you have all latest changes in this file 🙏🏻